api/GLES3/gl3.h

```diff
@@ -1,5 +1,5 @@
-#ifndef __gl3_h_
-#define __gl3_h_ 1
+#ifndef __gles2_gl3_h_
```

Comment:
Should this say "gles3_gl3_h"?

Reply:
Changing the 'apiname' to 'gles3' in genheaders.py also causes changes like no longer defining GL_ES_VERSION_2_0 in gl3.h. So that's not going to work. Note that despite the apparent inconsistency, this header guard does not clash with any others so it doesn't need fixing.

I think the way we need to interpret it is that 'apiname' and API version are completely different things. They might as well have called it 'glesa' and 'glesb' and each of them support different version numbers. Note also that OpenGL ES 3.x is recommended to be implemented in a libGLESv2 library.